spec/poros/food_spec.rb

```diff
@@ -0,0 +1,17 @@
+require "rails_helper"
+RSpec.describe Food do
+  it "exists" do
+    details = {
+      gtinUpc: "20042431",
+      description: "SWEET POTATO",
+      brandOwner: "FRESH & EASY",
+      ingredients: "SWEET POTATO, CHIPOTLE CHICKEN {COOKED CHICKEN (CHICKEN BREAST WITH RIB MEAT, CHICKEN BROTH, CONTAINS LESS THAN 2% OF: MODIFIED POTATO STARCH, GARLIC POWDER, CANOLA OIL, SOYBEAN OIL, SALT, SODIUM PHOSPHATE, CHICKEN FAT, FLAVOR), CHIPOTLE MANGO SAUCE [MANGO CHIPOTLE MARINADE (MANGO PUREE, WATER, CHIPOTLE PEPPER, ONION, SALT, BROWN SUGAR, CANOLA OIL, VINEGAR, GARLIC, LEMON JUICE CONCENTRATE, TOMATO PASTE, SPICES, RICE WINE VINEGAR, SUGAR, CILANTRO, CITRIC ACID)], WATER, CILANTRO}, MANGO SALSA (MANGO, CUCUMBER, TOMATO, LIME JUICE, MANGO PUREE, ONION, CILANTRO, SERRANO PEPPERS, SALT)."
+    }
```

Comment:
When creating these food objects, you are passing over a large chunk of json, so to test that your poro is accurately taking that json and making objects, you can copy and paste an example of the type of json this initialize method would be passed in the food poro.